defusedxml is not available on Solaris and it's not feasible
to bring it in just for testing purposes.

Not intended for upstream.

--- sphinx-8.2.0/tests/test_theming/test_theming.py
+++ sphinx-8.2.0/tests/test_theming/test_theming.py
@@ -8,7 +8,7 @@ from typing import TYPE_CHECKING
 from xml.etree.ElementTree import ParseError
 
 import pytest
-from defusedxml.ElementTree import parse as xml_parse
+from xml.etree.ElementTree import parse as xml_parse
 
 import sphinx.builders.html
 from sphinx.builders.html import StandaloneHTMLBuilder
--- sphinx-8.2.0/sphinx/testing/util.py
+++ sphinx-8.2.0/sphinx/testing/util.py
@@ -78,7 +78,7 @@ def assert_node(node: Node, cls: Any = N
 # keep this to restrict the API usage and to have a correct return type
 def etree_parse(path: str | os.PathLike[str]) -> ElementTree:
     """Parse a file into a (safe) XML element tree."""
-    from defusedxml.ElementTree import parse as xml_parse
+    from xml.etree.ElementTree import parse as xml_parse
 
     return xml_parse(path)
 
